processor/manager.go

```diff
@@ -32,25 +34,29 @@ func (proc *LifecycleManager) Run(ctx context.Context) error {
 // StartNew starts a processor, this is not a blocking call.
```

Comment:
documentation needs to be updated, at least the first word :)

Reply:
done